src/test/java/kitchenpos/application/TableServiceTest.java

```diff
@@ -0,0 +1,125 @@
+package kitchenpos.application;
+
+import static kitchenpos.Fixtures.빈테이블_1;
+import static kitchenpos.Fixtures.빈테이블_2;
+import static kitchenpos.Fixtures.주문_테이블1;
+import static kitchenpos.Fixtures.테이블_1;
+import static kitchenpos.Fixtures.테이블그룹;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import kitchenpos.dao.OrderDao;
+import kitchenpos.dao.OrderLineItemDao;
+import kitchenpos.dao.OrderTableDao;
+import kitchenpos.dao.TableGroupDao;
+import kitchenpos.domain.OrderTable;
+import kitchenpos.domain.TableGroup;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@SuppressWarnings("NonAsciiCharacters")
+class TableServiceTest extends ServiceTest {
+
+    @Autowired
+    TableService tableService;
+
+    @Autowired
+    OrderService orderService;
+
+    @Autowired
+    TableGroupDao tableGroupDao;
+
+    @Autowired
+    OrderTableDao orderTableDao;
+
+    @Autowired
+    OrderLineItemDao orderLineItemDao;
+
+    @Autowired
+    OrderDao orderDao;
+
+    @DisplayName("테이블을 생성할 수 있다.")
+    @Test
+    void create() {
+        OrderTable 테이블_1 = tableService.create(테이블_1());
+
+        assertThat(tableService.list())
+                .usingRecursiveFieldByFieldElementComparator()
+                .contains(테이블_1);
+    }
+
+    @DisplayName("테이블의 빈 상태 여부를 변경할 수 있다.")
+    @Test
+    void changeEmpty() {
+        OrderTable 테이블_1 = tableService.create(테이블_1());
+
+        테이블_1.updateEmpty(false);
+        tableService.changeEmpty(테이블_1.getId(), 테이블_1);
+
+        assertThat(orderTableDao.findById(테이블_1.getId()).orElseThrow().isEmpty())
+                .isFalse();
+    }
```

Comment:
```suggestion
                OrderTable orderTable = tableService.changeEmpty(테이블_1.getId(), 테이블_1);
                
                assertThat(orderTable.isEmpty()).isFalse();
            }
```

디비에서 조회하지 않고 이렇게도 작성할 수 있어보여요!

Reply:
오!! 수정해볼게요!!